packages/language-server/server/validation/typings.js

```diff
@@ -1,5 +1,5 @@
-const { getConfig } = require("@hegel/cli/lib/config");
-const { importModule } = require("@hegel/cli/lib/module");
+const { getConfig } = require("@hegel/cli/build/lib/config");
+const { importModule } = require("@hegel/cli/build/lib/module");
 const { getBabylonAST } = require("../utils/document-ast");
 const { createGlobalScope } = require("@hegel/core");
```

Comment:
`require("@hegel/core/build/index")` - you possibly forgot to add.

Reply:
no it's correct see https://github.com/JSMonk/hegel/blob/master/packages/core/package.json#L6